src/translate.rs

```diff
@@ -1162,38 +1176,64 @@ impl FromGlib<i32> for Option<u64> {
     }
 }
 
-/// Translate from a pointer type without taking ownership, transfer: none.
+/// Translate from a pointer type which is annotated with `transfer: none`.
+/// The resulting value is referenced at least once, by the bindings.
+/// The value must be unreferenced on `drop()`.
```

Comment:
s/must/will/ ?

Reply:
The trait can be defined on any type, so it can't guarantee that things will get dropped correctly. In other words, the trait cannot say "this trait will do this for you". The implementor is responsible for and must drop things, so I wrote "must" for "to implement this trait correctly, you must do this".

Reply:
You right

Reply:
I tried to make it a little more where the responsibility for `drop()` lies by spelling it out.